site/content/contribute/getting-started/_index.md

```diff
@@ -1,5 +1,4 @@
 ---
-draft: true
 title: "Getting Started"
```

Comment:
@jasonblais thoughts about making this page a lot simpler, similar to the integrate section?
https://developers.mattermost.com/integrate/getting-started/

Reply:
Great, I forgot we were already using that template. It'll be easy to migrate over to that.

Only question is: For step 2, I'm aiming to bring people right away to the project-specific contributor flow, with environment set up, etc. Should we keep that here, or how would we  go about it? 0/5

Reply:
Yeah I think keeping it here makes sense